src/components/accounts/AccountFormAccountId.js

```diff
@@ -128,6 +129,7 @@ class AccountFormAccountId extends Component {
     }))
 
     handleCheckAvailability = (accountId, type) => {
+        Mixpanel.track("CA Check account availability")
```

Comment:
You'll need to check `type === 'create'` if you're only tracking for create account since this component is used in a few other places too.
```

if (type === 'create') {
    Mixpanel.track("CA Check account availability")
} 
```

Reply:
sure, what other places are used this component?